AdyenComponents/OnlineBanking/OnlineBankingComponent.swift

```diff
@@ -0,0 +1,132 @@
+                                                                             style: configuration.style)
+
+    public var requiresModalPresentation: Bool = true
+
+    /// Component's configuration
+    public var configuration: Configuration
+
+    private let onlineBankingPaymentMethod: OnlineBankingPaymentMethod
+
+    private var selectedIssuer: Issuer
+
+    // MARK: - Items
+
+    private var termsAndConditionsLink: String {
+        return paymentMethod.type == .onlineBankingCZ ? "https://static.payu.com/sites/terms/files/payu_privacy_policy_cs.pdf" : "https://static.payu.com/sites/terms/files/payu_privacy_policy_sk.pdf"
+    }
+
+    /// The terms and condition message item.
+    // swiftlint:disable line_length
+    internal lazy var termsAndConditionsLabelItem: FormAttributedLabelItem = .init(termsAndConditionsText: localizedString(.onlineBankingTermsAndConditions, configuration.localizationParameters),
+                                                                      link: termsAndConditionsLink,
+                                                                      style: configuration.style.footnoteLabel,
+                                                                      linkTextStyle: configuration.style.linkTextLabel,
+                                                                      identifier: ViewIdentifierBuilder.build(scopeInstance: "AdyenDropIn.OnlineBankingComponent", postfix: "OnlineBankingTAndCLabel"))
+
+    /// The continue button item.
+    internal lazy var continueButton: FormButtonItem = {
+        let item = FormButtonItem(style: configuration.style.mainButtonItem)
+        item.identifier = ViewIdentifierBuilder.build(scopeInstance: "AdyenDropIn.OnlineBankingComponent", postfix: "continueButtonItem")
+        item.title = localizedString(.continueTitle, configuration.localizationParameters)
+        item.buttonSelectionHandler = { [weak self] in
+            self?.didSelectContinueButton()
+        }
+        return item
+    }()
+
+    /// The Issuer List item.
+    internal lazy var issuerListItem: FormIssuersPickerItem = {
+        let defaultIssuer = onlineBankingPaymentMethod.issuers[0]
+
+        let issuerListPickerItem: [IssuerPickerItem] = onlineBankingPaymentMethod.issuers.map({
+            return IssuerPickerItem(identifier: $0.identifier, element: $0)
+        })
+        let issuerPickerItem = FormIssuersPickerItem(preselectedValue: issuerListPickerItem[0],
+                                        selectableValues: issuerListPickerItem,
+                                        style: configuration.style.textField)
+        issuerPickerItem.title = localizedString(.selectFieldTitle, configuration.localizationParameters)
+        issuerPickerItem.identifier = ViewIdentifierBuilder.build(scopeInstance: "AdyenDropIn.OnlineBankingComponent", postfix: "issuersList")
+        _ = issuerPickerItem.publisher.addEventHandler({ issuerPickerITem in
```

Comment:
Would the `observe` function achieve the same thing here? `addEventHandler ` function is one step lower and is not used directly usually. You can check the usages of `observe`